barretenberg/cpp/src/barretenberg/stdlib/protogalaxy_verifier/protogalaxy_recursive_verifier.test.cpp

```diff
@@ -226,7 +227,11 @@ template <typename RecursiveFlavor> class ProtogalaxyRecursiveTests : public tes
         // Check for a failure flag in the recursive verifier circuit
 
         if constexpr (!IsSimulator<OuterBuilder>) {
+            // inefficiently check finalized size
+            folding_circuit.finalize_circuit();
+            info("Folding Recursive Verifier: num gates finalized = ", folding_circuit.num_gates);
```

Comment:
so this is now printing out finalized gates without the ensure nonzero gates? But in practice, our circuit is actually going to include these ensure_nonzero gates so I don't understand why we wouldn't want them.

Reply:
Yes--before the PR that was the behavior of this test, so I'm just preserving the test. Whether it _should_ be that behavior... I'll make an issue.

Reply:
https://github.com/AztecProtocol/barretenberg/issues/1100

Reply:
could link this to code but also no need if you just want to merge